src/encoding/text.rs

```diff
@@ -742,6 +747,51 @@ mod tests {
         parse_with_python_client(encoded);
     }
 
+    #[test]
+    fn encode_histogram_family_with_empty_family_labels() {
+        let mut registry = Registry::default();
+        let family =
+            Family::new_with_constructor(|| Histogram::new(exponential_buckets(1.0, 2.0, 10)));
+        registry.register("my_histogram", "My histogram", family.clone());
+
+        family
+            .get_or_create(&())
+            .observe(1.0);
+
+        let mut encoded = String::new();
+
+        encode(&mut encoded, &registry).unwrap();
+
+        parse_with_python_client(encoded);
+    }
+
+    #[test]
+    fn encode_histogram_family_with_empty_struct_family_labels() {
```

Comment:
Added this second test to give a more realistic use case of empty family labels.